README.md

```diff
@@ -223,6 +229,111 @@ The following code example allows push notifications to be displayed when your a
+
+In order for deep linking to work, there are a few configurations that are needed and these are no different from what are required for handling deep linking in general and [Apple documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) on this can be followed in conjunction with the steps highlighted here -
+
+### Step 1: Register the URL scheme
+
+In order for Apple to route a deep link to your application you need to register a URL scheme in your application's Info.plist file. This can be done using the editor that xcode provides from the Info tab of your project settings or by editing the Info.plist directly -
+
+The required fields are as following -
+
+1. **Identifier** - The identifier you supply with your scheme distinguishes your app from others that declare support for the same scheme. To ensure uniqueness, specify a reverse DNS string that incorporates your company’s domain and app name. Although using a reverse DNS string is a best practice, it doesn’t prevent other apps from registering the same scheme and handling the associated links.
+2. **URL schemes** - In the URL Schemes box, specify the prefix you use for your URLs.
+3. **Role** - Since your app will be editing the role select the role as editor
+
+In order to edit the Info.plist directly, just fill in your app specific details and paste this in your plist -
+
+```xml
+<key>CFBundleURLTypes</key>
+<array>
+	<dict>
+		<key>CFBundleTypeRole</key>
+		<string>Editor</string>
+		<key>CFBundleURLName</key>
+		<string>{your_unique_identifier}</string>
+		<key>CFBundleURLSchemes</key>
+		<array>
+			<string>{your_URL_scheme}</string>
+		</array>
+	</dict>
+</array>
+```
+
+
+### Step 2: Whitelist supported URL schemes
+
+Since iOS 9 Apple has mandated that the URL schemes that you app can open need to also be listed in the Info.plist. This is in addition to Step 1 above. Even if your app isn't opening any other apps, you still need to list your app's URL scheme in order for deep linking to work.
+
+This needs to be done in the Info.plist directly -
+
+```xml
+<key>LSApplicationQueriesSchemes</key>
+<array>
+	<string>{your custom URL scheme}</string>
+</array>
+```
+
+### Step 3: Implement handling deep links in your app
+
+Steps 1 & 2 set your app up for receiving deep links but now is when you need to figure out how to handle them within your app.
+
+If you are using UIKit, you need to implement [`application:openURL:options:`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application) in your application's app delegate.
+
+// TODO: add a link to the example app here once it has been moved to the SDL
```

Comment:
Do we need to take out this TODO, or are you holding this PR till the sample app is moved

Reply:
I thought I removed it, may be just after you added this comment but I'm now referencing the example app from here and it should go in next.